marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -215,93 +225,81 @@ instance ToJSON Utxo where
             , "txIndexInBlock"    .= (u ^. txIndexInBlock)
             ]
 
-data ChainPointRow
-    = ChainPointRow { _cpSlotNo :: C.SlotNo, _cpBlockHash :: C.Hash C.BlockHeader }
-    deriving (Show, Eq, Ord, Generic)
```

Comment:
Why did you delete it?

Reply:
This seemed to be an internal data type that added boilerplate, while there is no table for the chain point rows. I.e there didn't seem to be a reason to re-wrap the chain point, so I inlined it into the two places it was used.

Reply:
:+1: